source/tasks/Utils/tool.ts

```diff
@@ -52,9 +52,15 @@ export async function getOrInstallOctoCommandRunner(command: string) : Promise<E
 
     return resolvePublishedOctoVersion("latest")
     .then(getOrDownloadOcto)
+    .catch(err => {
+        tasks.error(err);
+        throw Error("Unable to locate and download the latest Octo command line tool. To use the embedded copy"
+            + " or another specific version, add the Octopus tools installer task to the build pipeline"
+            + " ahead of this task.");
```

Comment:
is `before` a better word? Not sure myself...

Reply:
Yeah probably. Changed.